sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/predicates.scala

```diff
@@ -487,7 +487,14 @@ case class In(value: Expression, list: Seq[Expression]) extends Predicate {
     }
   }
 
-  override def toString: String = s"$value IN ${list.mkString("(", ",", ")")}"
+  override def toString: String = {
+    val limit = SQLConf.get.optimizerInSetMemberLoggingLimit
```

Comment:
The name of the config confuses slightly. How is it related to "optimizer" and `Logging`?